addons/pos_restaurant/static/src/overrides/components/receipt_screen/receipt_screen.js

```diff
@@ -13,12 +13,6 @@ patch(ReceiptScreen.prototype, {
             }
         });
     },
-    //@override
-    _addNewOrder() {
-        if (!this.pos.config.module_pos_restaurant) {
-            super._addNewOrder(...arguments);
-        }
-    },
```

Comment:
I moved this logic in pos. `module_pos_restaurant` is a valid field in `point_of_sale` module.